modules/va_notify/spec/sidekiq/in_progress_form_reminder_spec.rb

```diff
@@ -42,25 +45,28 @@
         allow(VANotify::Veteran).to receive(:new).and_return(user_with_uuid)
 
         allow(VANotify::UserAccountJob).to receive(:perform_async)
-        expiration_date = in_progress_form.expires_at.strftime('%B %d, %Y')
+        expiration_date = in_progress_form_with_user_account_id.expires_at.strftime('%B %d, %Y')
 
         Sidekiq::Testing.inline! do
-          described_class.new.perform(in_progress_form.id)
+          described_class.new.perform(in_progress_form_with_user_account_id.id)
         end
 
-        expect(VANotify::UserAccountJob).to have_received(:perform_async).with('uuid', 'fake_template_id',
+        # rubocop:disable Layout/LineLength
+        expect(VANotify::UserAccountJob).to have_received(:perform_async).with(in_progress_form_with_user_account_id.user_account_id, 'fake_template_id',
                                                                                {
                                                                                  'first_name' => 'FIRST_NAME',
                                                                                  'date' => expiration_date,
                                                                                  'form_age' => ''
                                                                                })
+        # rubocop:enable Layout/LineLength
       end
     end
 
     describe 'multiple relevant in_progress_forms' do
       let!(:in_progress_form_1) do
         Timecop.freeze(7.days.ago)
-        in_progress_form = create(:in_progress_686c_form, user_uuid: user.uuid)
+        in_progress_form = create(:in_progress_686c_form, user_uuid: user.uuid,
+                                                          user_account_id: create(:user_account).id)
```

Comment:
non-blocking, but while you're updating formatting, what do you think of this instead? 
```suggestion
        in_progress_form = create(
          :in_progress_686c_form, user_uuid: user.uuid, user_account_id: create(:user_account).id
        )
```

Reply:
I like it well enough. Applied!